src/rules/no-enum.ts

```diff
@@ -9,7 +9,7 @@ const noEnum = createRule<Options, MessageIds>({
     type: "problem",
     docs: {
       description: "Disallow all types of TypeScript enums",
-      recommended: "error",
+      recommended: "recommended",
```

Comment:
TypeScript errored that this value was no longer allowed.

Options are now: `'recommended' | 'strict' | 'stylistic'`



https://github.com/typescript-eslint/typescript-eslint/blob/2a2a76ad6540177af19cbb4194895496978d288a/packages/utils/src/ts-eslint/Rule.ts#L20